docs_app/content/marketing/contributors.json

```diff
@@ -110,7 +110,7 @@
     "github": "https://github.com/mattpodwysocki",
     "picture": "https://avatars0.githubusercontent.com/u/49051",
     "twitter": "https://twitter.com/mattpodwysocki",
-    "group": "Alumn"
+    "group": "Alumni"
```

Comment:
As a non-native English speaker, I hope this one is right 🙂  I don't really know. Maybe @benlesh can help?